### DISCUSSION TITLE
node repeat next message output

### DISCUSSION THREAD

**Discussion by ghost** [jensrossbach/node-red-contrib-chronos#91] [General]
Hi,
can I get the value (time) of "next message at ...." in output?

![output](https://user-images.githubusercontent.com/105095607/182150858-cb5af2d3-312b-485b-a2d4-7f86466e046a.png)

**Comment by ghost** [comment]
I answer by myself ....
https://github.com/jensrossbach/node-red-contrib-chronos/discussions/74

But the option output port for event is not present in repeat node .... :cry:

**Reply by jensrossbach** [reply]
Yes, the event time output is currently only supported by the scheduler node, not the repeat node. If you require this functionality, please create [a feature request in the issue section](https://github.com/jensrossbach/node-red-contrib-chronos/issues/new/choose).

**Reply by jensrossbach** [reply]
Meanwhile I decided to not implement such functionality. However, it is possible to catch the whole status message with a status node and extract only the time portion from that string.
